usecases/modulecomponents/clients/jinaai/jinaai.go

```diff
@@ -0,0 +1,252 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package jinaai
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/weaviate/weaviate/entities/moduletools"
+
+	"github.com/weaviate/weaviate/usecases/modulecomponents"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	DefaultRPM = 500 // from https://jina.ai/embeddings/
+	DefaultTPM = 1_000_000
+)
```

Comment:
I'd leave those in the respective module - different endpoints often have different limit

Reply:
hmm, I can move it but here we are asking always the same `/v1/endpoint` endpoint

Reply:
ok, pushed changes